### ISSUE TITLE
inventory.audit reactor does not work with Salt 2019

### ISSUE THREAD

**Issue by neilmunday** [neilmunday/Salt-Minion-Inventory#14]
The inventory.audit reactor SLS fails to be parsed by the Salt master when using the 2019 version of Salt.

Error in Salt master logs:

```
2019-02-28 23:10:09,448 [salt.utils.reactor:106 ][ERROR   ][16095] Failed to render "/srv/reactor/inventory/audit.sls":
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/salt/utils/reactor.py", line 97, in render_reaction
    data=data)
  File "/usr/lib/python2.7/site-packages/salt/state.py", line 385, in render_template
    **kwargs)
  File "/usr/lib/python2.7/site-packages/salt/template.py", line 101, in compile_template
    ret = render(input_data, saltenv, sls, **render_kwargs)
  File "/usr/lib/python2.7/site-packages/salt/renderers/yaml.py", line 57, in render
    raise SaltRenderError(err_type, line_num, exc.problem_mark.buffer)
SaltRenderError: found unexpected ':'
```

This appears due to the _properties_ dictionary returned as part of the inventory.audit function.

**Comment by neilmunday** [comment]
A 2018 master works fine with a 2019 minion.

**Comment by neilmunday** [comment]
"tojson" required in Jinja code in reactor SLS files for to handle unicode dictionaries. See: https://docs.saltstack.com/en/latest/topics/releases/2019.2.0.html

Fixed as of commit: 0eb346eba8e4c9256228813a9fcbfed29047cbc4
